backend/app/tests/api/routes/test_tag.py

```
@@ -742,3 +742,131 @@ def test_delete_tag_by_id(
+
+
+def test_inactive_tagtype_not_listed(
+    client: TestClient,
+    db: SessionDep,
+    get_user_superadmin_token: dict[str, str],
+) -> None:
+    organization = setup_user_organization(db)
+    data = {
+        "name": "Inactive TagType Test",
+        "description": random_lower_string(),
+        "organization_id": organization.id,
```

Pylint (3.3.7)

[error] Instance of 'tuple' has no 'id' member

(E1101)